hydrus/tests/test_app.py

```diff
@@ -159,75 +162,83 @@ def test_Collections_PUT(self):
         index = self.client.get("/" + self.API_NAME)
         assert index.status_code == 200
         endpoints = json.loads(index.data.decode('utf-8'))
-        for collection_name in endpoints:
+        for endpoint in endpoints:
+            collection_name = "/".join(endpoints[endpoint].split(

```

Comment:
it is better to format URLs:
* or using `str.format()`: `'{}/{}/{}'.format(..., ..., ...)`
* or better string formatting: `f'{domain}/{path}/{id}'`

This is a wider problem in the codebase, and refactor should apply. I will open a issue for "General Refactoring".